streams/src/main/java/org/apache/kafka/streams/kstream/Materialized.java

```diff
@@ -39,15 +38,15 @@
  * by providing just a store name.
  */
 public class Materialized<K, V, S extends StateStore> {
-    protected StoreSupplier<S> storeSupplier;
```

Comment:
If you are removing the template `S` should it be removed from the declaration?

Reply:
Actually, I'm not clear why we'd want to remove `S` from the storeSupplier here?